api/v1/backup_types.go

```diff
@@ -237,14 +237,22 @@ type BackupStatus struct {
 	// +optional
 	Phase BackupPhase `json:"phase,omitempty"`
 
-	// When the backup was started
+	// When the backup execution was started by the backup tool
 	// +optional
 	StartedAt *metav1.Time `json:"startedAt,omitempty"`
 
-	// When the backup was terminated
+	// When the backup execution was terminated by the backup tool
 	// +optional
 	StoppedAt *metav1.Time `json:"stoppedAt,omitempty"`
 
+	// When the backup process was started by the operator
+	// +optional
+	ReconciliationStartedAt *metav1.Time `json:"reconciliationStartedAt,omitempty"`
+
+	// When the reconciliation was terminated by the operator (either successfully or not)
+	// +optional
+	ReconciliationTerminatedAt *metav1.Time `json:"reconciliationTerminatedAt,omitempty"`
```

Comment:
"terminated" jumps at me. It's not a word you use on success (unless the success is that the termination command was carried out).
So, for a term that works with either success or failure, how about "ended", "finished", "concluded"?
I particularly think "end" can also applied in the negative.

```suggestion
	// When the reconciliation was ended by the operator (either successfully or not)
	// +optional
	ReconciliationEndedAt *metav1.Time `json:"reconciliationTerminatedAt,omitempty"`
```